include/benchmark/benchmark.h

```diff
@@ -415,6 +415,13 @@ enum TimeUnit { kNanosecond, kMicrosecond, kMillisecond };
 // calculated automatically to the best fit.
 enum BigO { oNone, o1, oN, oNSquared, oNCubed, oLogN, oNLogN, oAuto, oLambda };
 
+// If OpenMP threading is used, two special values can be passed to Threads():
+// OmpDefaultNumThreads: use omp_get_max_threads().
+// OmpNoParallelRegion==1: don't open a parellel region. A parallel region with one thread is not possible.
```

Comment:
Can you please quote where the spec says that?
`#pragma omp parallel num_threads(1)` works just fine.

Reply:
The comment was misleading. I just meant, that my implementation didn't support parallel region with just one thread.
In a new version the issue will be gone by design.